### PR TITLE
Fixes #33664 - Adjust subman_setup snippet for debian

### PR DESCRIPTION
**Pull request by nadjaheitmann**

### CODE REVIEW

File: app/views/unattended/provisioning_templates/snippet/subscription_manager_setup.erb [hunk 1]
```diff
@@ -52,7 +54,9 @@ EOF
         subscription-manager clean
       fi
 
-      $PKG_MANAGER_REMOVE katello-ca-consumer\* > /dev/null 2>&1
+      if ![ -f /etc/debian_version ]; then
```
Comment (stejskalleos) [inline app/views/unattended/provisioning_templates/snippet/subscription_manager_setup.erb:57]: What's the reason to skip this removal on Debian?

Comment (m-bucher) [reply app/views/unattended/provisioning_templates/snippet/subscription_manager_setup.erb:57]: There are no deb-packages generated for the katello-ca-consumer. Therefore, the `katello-ca-consumer`-script is used to install certificates and adapt subscription-manager config.
Which means there is no package installed that can be removed here.

**Review by stejskalleos — APPROVED** [review]
:green_apple: **LGTM**

File: app/views/unattended/provisioning_templates/snippet/subscription_manager_setup.erb (hunk 1 above)
Comment (lhellebr) [reply app/views/unattended/provisioning_templates/snippet/subscription_manager_setup.erb:57]: This is a syntax error and caused:
```
bash: line 258: ![: command not found
```
There's a space missing after the exclamation mark.

Comment (nadjaheitmann) [reply app/views/unattended/provisioning_templates/snippet/subscription_manager_setup.erb:57]: @lhellebr Thanks for reporting. I am surprised that we did not stumble across this before https://github.com/theforeman/foreman/pull/10161

**Pull request merged**